src/library/base/man/is.finite.Rd

```diff
@@ -18,9 +18,10 @@
   and not missing) or infinite.
 
   \code{Inf} and \code{-Inf} are positive and negative infinity
-  whereas \code{NaN} means \sQuote{Not a Number}.  (These apply to numeric
-  values and real and imaginary parts of complex values but not to
-  values of integer vectors.)  \code{Inf} and \code{NaN} (as well as
+  whereas \code{NaN} means \sQuote{Not a Number}.  \code{Inf} and
+  \code{NaN} are class of numeric and type of double, and can be
```

Comment:
"are of class \code{numeric} and type \code{double}" ? (or quotes? not sure what the precise conventions are for when to use \code{} vs \sQuote{} vs ...)

Reply:
I decided to use \code as theyare also code and have their own doc pages